src/platform/packages/shared/kbn-visualization-ui-components/components/dimension_buttons/dimension_button.tsx

```diff
@@ -68,7 +68,7 @@ function DimensionButtonImpl({
         gap: ${euiTheme.size.s};
         min-height: ${euiTheme.size.xl};
         padding: ${euiTheme.size.xs} ${euiTheme.size.s};
-        background-color: ${euiTheme.colors.backgroundBaseSubdued};
+        background-color: ${euiTheme.colors.backgroundBaseHighlighted};
```

Comment:
can you please fix one of the 2 linked issues in the PR description, it seems to be connected to a PR rather then an actual issue.
I believe it should refer to this line change but not sure what was the original problem.
thanks